dashboard/src/components/Header/Header.test.tsx

```diff
@@ -163,3 +163,39 @@ it("doesn't call getNamespace when selecting all namespaces", () => {
   expect(setNamespace).toHaveBeenCalledWith("_all");
   expect(getNamespace).not.toHaveBeenCalled();
 });
+
+describe("ClusterSelector", () => {
+  it("does not render the cluster switcher when there is only one cluster", () => {
```

Comment:
I am not sure if we should hide the selector, it would help promoting the feature. In any case, I am not hiding it in the new UI.

Reply:
Yep, once the feature is released, I agree, but right now it'd be confusing to see a cluster selector and not be able to use it? (Fine for the new UI, which is also not released).

Reply:
yes, I mean once we release :+1: